release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1930,6 +1935,45 @@ With this release, the CCO no longer checks for the nonexistent permission.
+
+* Previously, Agent-based installations with dual-stack networking failed due to IPv6 connectivity check failures, even though IPv6 connectivity could be established between nodes. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-31631[*OCPBUGS-31631*])
+
+* Previously, due to a programming error, a script created compute server groups with the policy set for control planes. As a consequence, the 'serverGroupPolicy' property of `install-config.yaml` files was ignored for compute groups. With this fix, the server group policy set in the `install-config.yaml` file for compute machine pools is applied at installation in the script flow. (link:https://issues.redhat.com/browse/OCPBUGS-31050[*OCPBUGS-31050*])
+
+* Previously, when configuring an Agent-based installation that uses the `openshift-baremetal-install` binary, the installer erroneously attempted to verify the libvirt network interfaces. This might cause the following error: “Platform.BareMetal.externalBridge: Invalid value: "baremetal": could not find interface "baremetal"”. With this update, as the Agent-based installation method does not require libvirt, this erroneous validation has been disabled and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-30941[*OCPBUGS-30941*])
+
+* Previously, {product-title} upgrades for Ansible caused an error as the IPsec configuration was not idempotent. With this release, the issue is resolved. Now, all IPsec configurations for OpenShift Ansible playbooks are idempotent. (link:https://issues.redhat.com/browse/OCPBUGS-30802[*OCPBUGS-30802*])
+
+* Previously, using network types with dual-stack networking other than Open vSwitch-based software-defined networking (SDN) or Open Virtual Network (OVN) caused a validation error. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-30232[*OCPBUGS-30232*])
+
+* Previously, a closed IPv6 port range for `nodePort` services in user-provisioned-infrastructure installations on {rh-openstack} caused traffic through certain node ports to be blocked. With this release, appropriate security group rules have been added to the `security-group.yaml` playbook, resolving the issue. (link:https://issues.redhat.com/browse/OCPBUGS-30154[*OCPBUGS-30154*])
+
+* Previously, manifests that were generated by using the command ‘openshift-install agent create cluster-manifests’ command were not directly applied to an {product-titl} cluster because the manifests did not include type data. With this release, type data has been added to the manifests. Administrators can now apply the manifests to initiate a Zero Touch Provisioning (ZTP) installation that uses the same settings as the Agent-based installation. (link:https://issues.redhat.com/browse/OCPBUGS-29968[*OCPBUGS-29968*])
```

Comment:
```suggestion
* Previously, manifests that were generated by using the command `openshift-install agent create cluster-manifests` command were not directly applied to an {product-title} cluster because the manifests did not include type data. With this release, type data has been added to the manifests. Administrators can now apply the manifests to initiate a Zero Touch Provisioning (ZTP) installation that uses the same settings as the Agent-based installation. (link:https://issues.redhat.com/browse/OCPBUGS-29968[*OCPBUGS-29968*])
```

Reply:
:heavy_check_mark: